index.js

```diff
@@ -56,10 +57,14 @@ module.exports = function(options) {
       podVariation = options.modulePrefix + parsedName.fullNameWithoutType.slice(11) + '/' + parsedName.type;
       withinComponentPodVariation = options.modulePrefix + 'components/' + parsedName.fullNameWithoutType.slice(11) + '/' + parsedName.type;
     }
+
+    var withinPodModulePrefixVariation = "./" + options.podModulePrefix + podVariation.slice(1);
```

Comment:
Hopefully the naming isn't too confusing. `withinPodModulePrefixVariation` indicates components that are located within the directory specified by `podModulePrefix`.